lib/workers/branch/check-existing.ts

```diff
@@ -22,6 +22,15 @@ export async function prAlreadyExisted(
   if (pr) {
     logger.debug('Found closed PR with current title');
     const prDetails = await platform.getPr(pr.number);
+
+    const renovateIgnoreLabel = prDetails.labels?.indexOf('renovate-ignore');
```

Comment:
I think the label should be configurable

Reply:
It was mentioned in the issue - https://github.com/renovatebot/renovate/issues/10674 - that for now it could be a fixed label -"renovate-ignore"

Reply:
Can you make it at least a module const?